src/Controller/DataFlowJobController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of itk-dev/datatidy.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace App\Controller;
+
+use App\Entity\DataFlowJob;
+use App\Repository\DataFlowJobRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/data/flow2/job")
+ */
+class DataFlowJobController extends AbstractController
+{
+    /**
+     * @Route("/", name="data_flow_job_index", methods={"GET"})
+     */
+    public function index(DataFlowJobRepository $dataFlowJobRepository): Response
+    {
+        return $this->render('data_flow_job/index.html.twig', [
+            'data_flow_jobs' => $dataFlowJobRepository->findAll(),
```

Comment:
Should we limit this to runs for flows that are editable by the current user?

Reply:
Done.